Add the omicverse package

Starlitnightly opened the pull request:
Add the omicverse package.

(See https://github.com/Starlitnightly/omicverse/ for details.)

OmicVerse is the fundamental package for multi omics included bulk and single cell analysis with Python. For more information, please read our paper: [OmicVerse: A single pipeline for exploring the entire transcriptome universe](https://www.biorxiv.org/content/10.1101/2023.06.06.543913v1)

The original name of the omicverse was [Pyomic](https://pypi.org/project/Pyomic/), but we wanted to address a whole universe of transcriptomics, so we changed the name to OmicVerse, it aimed to solve all task in RNA-seq.

BulkTrajBlend algorithm in OmicVerse that combines Beta-Variational AutoEncoder for deconvolution and graph neural networks for overlapping community discovery to effectively interpolate and restore the continuity of “interrupted” cells in the original scRNA-seq data.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

bgruening commented on recipes/omicverse/meta.yaml:
```diff
@@ -0,0 +1,67 @@
+  run:
+    - python >=3.8
+    - numpy ==1.23.5
+    - pandas ==1.5.3
```
Are those strict pins really necessary?

Starlitnightly replied:
Thanks for your comment, we set numpy >=1.22,<1.24, pandas <=1.5.3  could run successfully in new pr too. 
- 1.5.3 is the last version of `pandas` 1.0, the next version is 2.0 and a large number of api's have been rewritten leaving many of the associated dependencies unsupported. 
- `numpy` 1.24 also changed some of the underlying api names accordingly, so the original dependency limit was lower than 1.23 and higher than 1.22. To ensure the smooth operation of the omicverse, we restricted the version of numpy.

Starlitnightly commented:
@bgruening Dear reviewer,

Thank you for taking the time to review my pull request and provide valuable feedback. I have addressed your suggestions and resolved all the issues. I believe these changes will enhance the quality and functionality of the code.

I kindly request you to review my pull request once again and merge my changes into the main repository. I am confident that these modifications will have a positive impact on the project's development.

If you have any further questions or suggestions, please feel free to ask. I am eager to further improve and refine the code.

Thank you again for your patience and support!

Best regards,
Zehua
